.travis.yml

```diff
@@ -134,6 +134,8 @@ matrix:
           env:
             - TOXENV=linux_warnings
 
-install: bash .travis/install.sh
+install:
+  - bash .travis/install.sh
+  - python -c "from scapy.all import conf; print(repr(conf))"
```

Comment:
Is this needed?

Reply:
It's not shown by default (requires to be clicked on), and will help to debug a lot of things.
I'd say it's fine to leave it